GWT build fails

Using the following versions:
libGdx version: 1.9.10
ashley version: 1.7.3
gwt gradle plugin version: 1.0.9
gwt version: 2.8.2

The project build fails with the following error:
```
Compiling module com.appelit.game_base.GdxDefinition
   Tracing compile failure path for type 'com.badlogic.ashley.utils.ImmutableArray'
      [ERROR] Errors in 'com/badlogic/ashley/utils/ImmutableArray.java'
         [ERROR] Line 74: Type mismatch: cannot convert from capture#1-of ?[] to V[]
   [ERROR] Aborting compile due to errors in some input files
```

My best guess is that this is caused because the method argument is of type `Class<?>` instead of `Class<V>` but I do not have experience with GWT so if anybody can shed some light on the issue it would be appreciated...

--- a/ashley/src/com/badlogic/ashley/utils/ImmutableArray.java
+++ b/ashley/src/com/badlogic/ashley/utils/ImmutableArray.java
@@ -70,7 +70,7 @@ public class ImmutableArray<T> implements Iterable<T> {
 		return array.toArray();
 	}
 
-	public <V> V[] toArray (Class<?> type) {
+	public <V> V[] toArray (Class<V> type) {
 		return array.toArray(type);
 	}
 	

